src/build-index.js

```diff
@@ -96,8 +96,26 @@ const specs = require("../specs.json")
 // - final `spec` ensures that properties defined in specs.json override info
 // from the source.
 fetchInfo(specs, { w3cApiKey })
-  .then(specInfo => specs.map(spec =>
-    Object.assign({}, spec, specInfo[spec.shortname], spec)))
+  .then(specInfo => specs.map(spec => {
+    const res = Object.assign({}, spec, specInfo[spec.shortname], spec);
```

Comment:
now that we don't just return `Object.assign`, the comment describing it above should be moved in-line.